controllers/disruption_controller.go

```diff
@@ -744,7 +638,7 @@ func (r *DisruptionReconciler) handleOrphanedChaosPods(req ctrl.Request) error {
 // the pod target will be moved to ignored targets, so it is not picked up by the next reconcile loop
 func (r *DisruptionReconciler) handleChaosPodsTermination(instance *chaosv1beta1.Disruption) error {
 	// get already existing chaos pods for the given disruption
-	chaosPods, err := r.getChaosPods(instance, nil)
+	chaosPods, err := r.ChaosPodService.GetChaosPodsOfDisruption(context.Background(), instance, nil)
```

Comment:
Let's get a context from the caller and use that here?

Reply:
Good idea 👍